include/fcntl.h

```diff
@@ -95,6 +95,7 @@
 #define F_SETLKW    12 /* Like F_SETLK, but wait for lock to become available */
 #define F_SETOWN    13 /* Set pid that will receive SIGIO and SIGURG signals for fd */
 #define F_SETSIG    14 /* Set the signal to be sent */
+#define F_GETPATH   15 /* Get the path of the file descriptor(macos) */
```

Comment:
A curious question: why is `macos` stated in the description?

Reply:
F_GETPATH is BSD/macOS specific:
https://lwn.net/Articles/277736/

Reply:
Update to (BSD/macOS)